graphics/graphite2/Portfile

```diff
@@ -17,15 +18,13 @@ long_description    Graphite is a “smart font” system developed specifically
                     to handle the complexities of lesser-known languages of \
                     the world.
 
-homepage            http://graphite.sil.org/
-master_sites        sourceforge:project/silgraphite/graphite2
-extract.suffix      .tgz
+homepage            https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home
 
-checksums           rmd160  cc69347fcc1a7cb2b332377e51d652ccac5d712c \
-                    sha256  ec0185b663059553fd46e8c4a4f0dede60a02f13a7a1fefc2ce70332ea814567
+checksums           rmd160  f5d082e3a3c1dd5fbf18b5ef32044222875171f9 \
+                    sha256  4739d16bef8e98b96d67f79003de492deff1c6924df58a30ff1902e2a3c6c45c \
+                    size    6667476
 
-patchfiles          patch-src-CMakeLists.txt.diff \
-                    patch-src-Face.cpp.diff
+cmake.out_of_source yes
```

Comment:
Is there a reason not to just use the cmake 1.1 portgroup instead where this is the default?